styles/deepl/catppuccin.user.css

```diff
@@ -182,10 +182,6 @@
     color: darken(@accent-color, 15%);
   }
 
-  [class*="bg-[#B4DAE8]"] {
-    // background-color: fadeout(@accent-color, 70%);
-  }
-
```

Comment:
Not sure if this was a note-to-self or something. There's another reference to the same selector on line 194/195 with another commented out `background-color` rule as well.